src/components/searchInput/searchInput.tsx

```diff
@@ -0,0 +1,29 @@
+import './_searchInput.scss'
+import SearchIcon from '@mui/icons-material/Search';
+
+interface SearchInputProps {
```

Comment:
It may be better to put types in a separate file for types

Reply:
You are right, I have corrected it.